test/pls.jl

```diff
@@ -31,7 +31,7 @@ include("modelcache.jl")
     @test_logs (:warn, "Model has not been fit") show(fm1)
 
     @test objective(updateL!(setθ!(fm1, [0.713]))) ≈ 327.34216280955366
-    @test_deprecated MixedModels.describeblocks(IOBuffer(), fm1)
+    #@test_deprecated MixedModels.describeblocks(IOBuffer(), fm1)
```

Comment:
If you want to remove the `describeblocks` deprecation machinery as well, go for it! This will be the second major release after that deprecation.

Reply:
Okay, let's nuke `describeblocks` then.  Do you want to do it or shall I?

Reply:
I'll let you do it.

Reply:
Done in https://github.com/JuliaStats/MixedModels.jl/pull/464/commits/3ee812d63fa74da7a9fe8f3b3150b2794801d4c7